### PR TITLE
Open and Accessible Data Tiering blog

### PR DESCRIPTION
**Pull request by EvanHStanton**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

File: src/content/blog/2024-09-04-community-tiering/index.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                                      Foreign table "public.people"
+ Column |  Type   | Collation | Nullable | Default | FDW options  | Storage  | Stats target | Description
+--------+---------+-----------+----------+---------+--------------+----------+--------------+-------------
+ name   | text    |           | not null |         | (key 'true') | extended |              |
+ age    | numeric |           | not null |         | (key 'true') | main     |              |
+Server: pg_tier_s3_srv
+FDW options: (dirname 's3://cdb-plat-use1-prod-instance-storage/v2/reservedly-lovable-sheepdog/public_people/')
+```
+
+Every Tembo Cloud instance has it's own private and dedicated S3 bucket.
```
Comment (bonesmoses) [inline src/content/blog/2024-09-04-community-tiering/index.mdx:86]: Should be "its own"

File: src/content/blog/2024-09-04-community-tiering/index.mdx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+Create an example table, and give it some data
+
+```sql
+CREATE table people (
+    name text not null,
+    age numeric not null
+);
+```
+```sql
```
Comment (bonesmoses) [inline src/content/blog/2024-09-04-community-tiering/index.mdx:61]: Why two separate blocks?

File: src/content/blog/2024-09-04-community-tiering/index.mdx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+	'AWS_REGION'
+);
+``
+```sql
+select tier.table('my-table');
+```
+
+```sql
+\d people
+```
```
Comment (bonesmoses) [inline src/content/blog/2024-09-04-community-tiering/index.mdx:108]: These could all be one contiguous SQL block. White space is allowed inside these if that's what you're trying to do.

**Comment by EvanHStanton** [comment]
Closing in favor of #654

**Pull request closed without merging**